st2common/st2common/models/db/execution.py

```diff
@@ -98,7 +98,7 @@ class ActionExecutionDB(stormbase.StormFoundationDB):
 
     def get_uid(self):
         # TODO Construct od from non id field:
-        uid = [self.RESOURCE_TYPE, str(self.id)]
+        uid = [self.RESOURCE_TYPE, str(self.id)]  # pylint: disable=no-member
```

Comment:
Please add a comment on why we need this pylint pragma and hopefully we can figure out root cause in the near future.

Reply:
I don't know the root cause. pylint just randomly decided to not like that line, so I dug through the code, and checked the mongoengine documentation to make sure that the object would have an `id` attribute by the time that code is evaluated.